Munch task87

xulean opened the pull request:
added 126 images

xulean committed:
Delete brownie.jpg

xulean committed:
new file:   frontend/src/assets/images/BBQ_Pork_Bun.jpg
new file:   frontend/src/assets/images/Baja_Burrito.jpg
new file:   frontend/src/assets/images/Beef_Broccoli.jpg
new file:   frontend/src/assets/images/Beef_Dumpling.jpg
new file:   frontend/src/assets/images/Beef_Hand-Ripped_Noodle_with_Special_Sauce.jpg
new file:   frontend/src/assets/images/Beef_Pancake_Roll.jpg
new file:   frontend/src/assets/images/Beef_Pot_Sticker.jpg
new file:   frontend/src/assets/images/Beef_Soup.jpg
new file:   frontend/src/assets/images/Beef_with_Pickled_Noodle_Soup.jpg
new file:   frontend/src/assets/images/Braised_Beef_Noodle_Soup.jpg
new file:   frontend/src/assets/images/Braised_Beef_with_Pickled_Noodle_Soup.jpg
new file:   frontend/src/assets/images/Breakfast_Burrito.jpg
new file:   frontend/src/assets/images/Breakfast_Plate.jpg
new file:   frontend/src/assets/images/Breakfast_Quesadilla.jpg
new file:   frontend/src/assets/images/Brownie_Sundae.jpg
new file:   frontend/src/assets/images/Caesar_Salad.jpg
new file:   frontend/src/assets/images/California_Scramble.jpg
new file:   frontend/src/assets/images/Cantina_Tacos.jpg
new file:   frontend/src/assets/images/Chicken_Chow_Mein.jpg
new file:   frontend/src/assets/images/Chicken_Dumpling.jpg
new file:   frontend/src/assets/images/Chicken_Fried_Rice.jpg
new file:   frontend/src/assets/images/Chicken_Hand-Ripped_Noodle_with_Special_Sauce.jpg
new file:   frontend/src/assets/images/Chicken_Mushroom_Noodle_Soup.jpg
new file:   frontend/src/assets/images/Chicken_Tenders.jpg
new file:   frontend/src/assets/images/Chile_Relleno.jpg
new file:   frontend/src/assets/images/Chile_Relleno_Burrito.jpg
new file:   frontend/src/assets/images/Chongqing_Style_Fried_Spicy_Chicken.jpg
new file:   frontend/src/assets/images/Chopped_Salad.jpg
new file:   frontend/src/assets/images/Churro.jpg
new file:   frontend/src/assets/images/Classic_Burrito.jpg
new file:   frontend/src/assets/images/Classic_Taco.jpg
new file:   frontend/src/assets/images/Cold_Noodle_with_Sesame_Sauce.jpg
new file:   frontend/src/assets/images/Crispy_Fish_Taco.jpg
new file:   frontend/src/assets/images/Cucumber_with_Garlic_Sauce.jpg
new file:   frontend/src/assets/images/Curry_Chicken_Rice_Bowl.jpg
new file:   frontend/src/assets/images/Dan_Dan_Noodle.jpg
new file:   frontend/src/assets/images/Del_Mar_Burrito.jpg
new file:   frontend/src/assets/images/Egg_roll.jpg
new file:   frontend/src/assets/images/Eggplant_with_Garlic_Sauce.jpg
new file:   frontend/src/assets/images/Empanadas.jpg
new file:   frontend/src/assets/images/Enchilada_Burrito.jpg
new file:   frontend/src/assets/images/Enchiladas.jpg
new file:   frontend/src/assets/images/Fajita_Taco.jpg
new file:   frontend/src/assets/images/Fajitas.jpg
new file:   frontend/src/assets/images/Fajitas_Burrito.jpg
new file:   frontend/src/assets/images/Flauta.jpg
new file:   frontend/src/assets/images/Fried_Cruller.jpg
new file:   frontend/src/assets/images/Fried_Rice.jpg
new file:   frontend/src/assets/images/Garden_Salad.jpg
new file:   frontend/src/assets/images/Green_Onion_Pancake.jpg
new file:   frontend/src/assets/images/Grill_Fish_Taco.jpg
new file:   frontend/src/assets/images/Home_Style_Chow_Mein.jpg
new file:   frontend/src/assets/images/Home_Style_Fried_Rice.jpg
new file:   frontend/src/assets/images/House_Special_Beef_Noodle_Soup.jpg
new file:   frontend/src/assets/images/House_Special_Burrito.jpg
new file:   frontend/src/assets/images/Huevo_Ranchero.jpg
new file:   frontend/src/assets/images/Ice_Cream.jpg
new file:   frontend/src/assets/images/Juicy_Pork_Dumpling_With_Peanuts.jpg
new file:   frontend/src/assets/images/Kids_Bean_and_Cheese_Burrito.jpg
new file:   frontend/src/assets/images/Kids_Breakfast_Burrito.jpg
new file:   frontend/src/assets/images/Kids_Cheese_Quesadilla.jpg
new file:   frontend/src/assets/images/Kids_Chicken_Tenders.jpg
new file:   frontend/src/assets/images/Kids_Classic_Taco.jpg
new file:   frontend/src/assets/images/Kids_Crispy_Rolled_Taco.jpg
new file:   frontend/src/assets/images/Kung_Pao_Chicken.jpg
new file:   frontend/src/assets/images/Kung_Pao_Shrimp_with_Peanuts.jpg
new file:   frontend/src/assets/images/Mapo_Tofu.jpg
new file:   frontend/src/assets/images/Mapo_Tofu_Rice_Bowl.jpg
new file:   frontend/src/assets/images/Mexican_Paletas.jpg
new file:   frontend/src/assets/images/Mongolian_Beef.jpg
new file:   frontend/src/assets/images/Nachos.jpg
new file:   frontend/src/assets/images/Noodle_And_Pork_Vegetable_with_Garlic_Sauce.jpg
new file:   frontend/src/assets/images/Noodle_with_Black_Bean_Sauce.jpg
new file:   frontend/src/assets/images/Noodle_with_Tomato_and_Egg.jpg
new file:   frontend/src/assets/images/Orange_Chicken.jpg
new file:   frontend/src/assets/images/Original_Flavored_Fried_Chicken.jpg
new file:   frontend/src/assets/images/Papas.jpg
new file:   frontend/src/assets/images/Popcorn_Chicken.jpg
new file:   frontend/src/assets/images/Pork_Chow_Mein.jpg
new file:   frontend/src/assets/images/Pork_Chow_Mein_with_Hot_Chili.jpg
new file:   frontend/src/assets/images/Pork_Chow_Mein_with_Spicy_Sauce.jpg
new file:   frontend/src/assets/images/Pork_Chow_Mein_with_Vinegar_Sauce.jpg
new file:   frontend/src/assets/images/Pork_Hand-Ripped_Noodle_with_Special_Sauce.jpg
new file:   frontend/src/assets/images/Pork_Pot_Sticker.jpg
new file:   frontend/src/assets/images/Pork_Vegetable_with_Garlic_Sauce_Rice_Bowl.jpg
new file:   frontend/src/assets/images/Pork_and_Napa_Dumpling.jpg
new file:   frontend/src/assets/images/Quesadilla.jpg
new file:   frontend/src/assets/images/Red_Bean_Bun.jpg
new file:   frontend/src/assets/images/Rice.jpg
new file:   frontend/src/assets/images/Santa_Barbara_Wrap.jpg
new file:   frontend/src/assets/images/Seafood_Noodle_Soup.jpg
new file:   frontend/src/assets/images/Shrimp_Broccoli.jpg
new file:   frontend/src/assets/images/Shrimp_Chow_Mein.jpg
new file:   frontend/src/assets/images/Shrimp_Cocktail.jpg
new file:   frontend/src/assets/images/Shrimp_Fried_Rice.jpg
new file:   frontend/src/assets/images/Shrimp_Hand-Ripped_Noodle_with_Special_Sauce.jpg
new file:   frontend/src/assets/images/Shrimp_Taco.jpg
new file:   frontend/src/assets/images/Shrimp_and_Chive_Egg_Dumpling.jpg
new file:   frontend/src/assets/images/Side_Salad.jpg
new file:   frontend/src/assets/images/Slider.jpg
new file:   frontend/src/assets/images/Sopa_Del_Dia.jpg
new file:   frontend/src/assets/images/Sopes.jpg
new file:   frontend/src/assets/images/Spanish_Omelette.jpg
new file:   frontend/src/assets/images/Spicy_Beef.jpg
new file:   frontend/src/assets/images/Stir-Fried_Baby_Bok_Choy.jpg
new file:   frontend/src/assets/images/Stir-Fried_Green_Beans.jpg
new file:   frontend/src/assets/images/Summer_Wrap.jpg
new file:   frontend/src/assets/images/TG_Breakfast_Torta.jpg
new file:   frontend/src/assets/images/TG_Burger.jpg
new file:   frontend/src/assets/images/TG_Burrito.jpg
new file:   frontend/src/assets/images/TG_Cubano.jpg
new file:   frontend/src/assets/images/TG_Nachos.jpg
new file:   frontend/src/assets/images/Taquitos.jpg
new file:   frontend/src/assets/images/Tomato_and_Egg_Rice_Bowl.jpg
new file:   frontend/src/assets/images/Torta.jpg
new file:   frontend/src/assets/images/Tortillas_Q.jpg
new file:   frontend/src/assets/images/Tortillas_Salad.jpg
new file:   frontend/src/assets/images/Vegetable_Chow_Mein.jpg
new file:   frontend/src/assets/images/Vegetable_Dumpling.jpg
new file:   frontend/src/assets/images/Vegetable_Fried_Rice.jpg
new file:   frontend/src/assets/images/Vegetable_Noodle_Soup.jpg
new file:   frontend/src/assets/images/Vegetables_Hand-Ripped_Noodle_with_Special_Sauce.jpg
new file:   frontend/src/assets/images/Veggie_Burrito.jpg
new file:   frontend/src/assets/images/Veggie_Taco.jpg
new file:   frontend/src/assets/images/Wonton_Soup.jpg
new file:   frontend/src/assets/images/Wonton_with_Chili_Oil.jpg
new file:   frontend/src/assets/images/Zucchnini.jpg
new file:   frontend/src/assets/images/brownie.jpg

xulean committed:
deleted:    frontend/.gitignore
	deleted:    frontend/package-lock.json

xulean committed:
deleted:    backend/.gitignore
	deleted:    backend/package-lock.json
	deleted:    package-lock.json

xulean committed:
added images

added images (125 images)

xulean commented:
please review it

kelseyp commented on backend/.gitignore:
Please do not delete the .gitignore

kelseyp commented on frontend/.gitignore:
Please do not delete the .gitignore

kelseyp commented on backend/package-lock.json:
There should be no reason to delete the package-lock.json

kelseyp commented on frontend/package-lock.json:
There should be no reason to delete the package-lock.json

kelseyp committed:
Revert "	deleted:    frontend/.gitignore"

This reverts commit 4b11f007e2820ea10fa6caccaeb6804cceae6133.

kelseyp committed:
Revert "	deleted:    backend/.gitignore"

This reverts commit 2450b35cbd2688695a26dbe9d857e7bee94cbfdb.

kelseyp committed:
Fixed missmatch spellings of image names

kelseyp committed:
Removed unnecessary package-lock. Updated all other lock files

xulean committed:
modified:   frontend/src/assets/images/Baja_Burrito.jpg
modified:   frontend/src/assets/images/Beef_Broccoli.jpg
modified:   frontend/src/assets/images/Beef_Dumpling.jpg
modified:   frontend/src/assets/images/Beef_Hand-Ripped_Noodle_with_Special_Sauce.jpg
modified:   frontend/src/assets/images/Beef_Pancake_Roll.jpg
modified:   frontend/src/assets/images/Beef_Pot_Sticker.jpg
modified:   frontend/src/assets/images/Beef_Soup.jpg
modified:   frontend/src/assets/images/Beef_with_Pickled_Noodle_Soup.jpg
modified:   frontend/src/assets/images/Braised_Beef_Noodle_Soup.jpg
modified:   frontend/src/assets/images/Braised_Beef_with_Pickled_Noodle_Soup.jpg
modified:   frontend/src/assets/images/Breakfast_Burrito.jpg
modified:   frontend/src/assets/images/Breakfast_Plate.jpg
modified:   frontend/src/assets/images/Breakfast_Quesadilla.jpg
new file:   frontend/src/assets/images/Brownie (2).jpg
modified:   frontend/src/assets/images/Caesar_Salad.jpg
modified:   frontend/src/assets/images/Cantina_Tacos.jpg
modified:   frontend/src/assets/images/Chicken_Chow_Mein.jpg
modified:   frontend/src/assets/images/Chicken_Dumpling.jpg
modified:   frontend/src/assets/images/Chicken_Fried_Rice.jpg
modified:   frontend/src/assets/images/Chicken_Hand-Ripped_Noodle_with_Special_Sauce.jpg
modified:   frontend/src/assets/images/Chicken_Mushroom_Noodle_Soup.jpg
modified:   frontend/src/assets/images/Chile_Relleno.jpg
modified:   frontend/src/assets/images/Chile_Relleno_Burrito.jpg
modified:   frontend/src/assets/images/Chongqing_Style_Fried_Spicy_Chicken.jpg
modified:   frontend/src/assets/images/Chopped_Salad.jpg
modified:   frontend/src/assets/images/Churro.jpg
modified:   frontend/src/assets/images/Classic_Burrito.jpg
modified:   frontend/src/assets/images/Classic_Taco.jpg
modified:   frontend/src/assets/images/Cold_Noodle_with_Sesame_Sauce.jpg
modified:   frontend/src/assets/images/Crispy_Fish_Taco.jpg
modified:   frontend/src/assets/images/Cucumber_with_Garlic_Sauce.jpg
modified:   frontend/src/assets/images/Curry_Chicken_Rice_Bowl.jpg
modified:   frontend/src/assets/images/Dan_Dan_Noodle.jpg
modified:   frontend/src/assets/images/Del_Mar_Burrito.jpg
modified:   frontend/src/assets/images/Egg_roll.jpg
modified:   frontend/src/assets/images/Eggplant_with_Garlic_Sauce.jpg
modified:   frontend/src/assets/images/Empanadas.jpg
modified:   frontend/src/assets/images/Enchiladas.jpg
modified:   frontend/src/assets/images/Fajita_Taco.jpg
modified:   frontend/src/assets/images/Fajitas.jpg
modified:   frontend/src/assets/images/Fajitas_Burrito.jpg
modified:   frontend/src/assets/images/Flauta.jpg
modified:   frontend/src/assets/images/Fried_Cruller.jpg
modified:   frontend/src/assets/images/Fried_Rice.jpg
modified:   frontend/src/assets/images/Garden_Salad.jpg
modified:   frontend/src/assets/images/Garlic_Flavored_Fried_Chicken.jpg
modified:   frontend/src/assets/images/Green_Onion_Pancake.jpg
modified:   frontend/src/assets/images/Grill_Fish_Taco.jpg
modified:   frontend/src/assets/images/Home_Style_Fried_Rice.jpg
modified:   frontend/src/assets/images/House_Special_Beef_Noodle_Soup.jpg
modified:   frontend/src/assets/images/House_Special_Burrito.jpg
modified:   frontend/src/assets/images/Huevo_Ranchero.jpg
modified:   frontend/src/assets/images/Ice_Cream.jpg
modified:   frontend/src/assets/images/Juicy_Pork_Dumpling_With_Peanuts.jpg
modified:   frontend/src/assets/images/Kids_Bean_and_Cheese_Burrito.jpg
modified:   frontend/src/assets/images/Kids_Breakfast_Burrito.jpg
modified:   frontend/src/assets/images/Kids_Cheese_Quesadilla.jpg
modified:   frontend/src/assets/images/Kids_Chicken_Tenders.jpg
modified:   frontend/src/assets/images/Kids_Classic_Taco.jpg
modified:   frontend/src/assets/images/Kids_Crispy_Rolled_Taco.jpg
modified:   frontend/src/assets/images/Kung_Pao_Chicken.jpg
modified:   frontend/src/assets/images/Mapo_Tofu.jpg
modified:   frontend/src/assets/images/Mapo_Tofu_Rice_Bowl.jpg
modified:   frontend/src/assets/images/Mexican_Paletas.jpg
modified:   frontend/src/assets/images/Mongolian_Beef.jpg
new file:   frontend/src/assets/images/Mozzarella_Sicks.jpg
modified:   frontend/src/assets/images/Nachos.jpg
modified:   frontend/src/assets/images/Noodle_And_Pork_Vegetable_with_Garlic_Sauce.jpg
modified:   frontend/src/assets/images/Noodle_with_Black_Bean_Sauce.jpg
modified:   frontend/src/assets/images/Noodle_with_Tomato_and_Egg.jpg
modified:   frontend/src/assets/images/Orange_Chicken.jpg
modified:   frontend/src/assets/images/Original_Flavored_Fried_Chicken.jpg
modified:   frontend/src/assets/images/Papas.jpg
modified:   frontend/src/assets/images/Popcorn_Chicken.jpg
modified:   frontend/src/assets/images/Pork_Chow_Mein.jpg
modified:   frontend/src/assets/images/Pork_Chow_Mein_with_Hot_Chili.jpg
modified:   frontend/src/assets/images/Pork_Chow_Mein_with_Spicy_Sauce.jpg
modified:   frontend/src/assets/images/Pork_Chow_Mein_with_Vinegar_Sauce.jpg
modified:   frontend/src/assets/images/Pork_Fried_Rice.jpg
modified:   frontend/src/assets/images/Pork_Hand-Ripped_Noodle_with_Special_Sauce.jpg
modified:   frontend/src/assets/images/Pork_Vegetable_with_Garlic_Sauce_Rice_Bowl.jpg
modified:   frontend/src/assets/images/Pork_and_Napa_Dumpling.jpg
modified:   frontend/src/assets/images/Quesadilla.jpg
modified:   frontend/src/assets/images/Red_Bean_Bun.jpg
modified:   frontend/src/assets/images/Santa_Barbara_Wrap.jpg
modified:   frontend/src/assets/images/Seafood_Noodle_Soup.jpg
modified:   frontend/src/assets/images/Shrimp_Broccoli.jpg
modified:   frontend/src/assets/images/Shrimp_Chow_Mein.jpg
modified:   frontend/src/assets/images/Shrimp_Cocktail.jpg
modified:   frontend/src/assets/images/Shrimp_Hand-Ripped_Noodle_with_Special_Sauce.jpg
modified:   frontend/src/assets/images/Shrimp_and_Chive_Egg_Dumpling.jpg
modified:   frontend/src/assets/images/Side_Salad.jpg
modified:   frontend/src/assets/images/Slider.jpg
modified:   frontend/src/assets/images/Sopa_Del_Dia.jpg
modified:   frontend/src/assets/images/Spanish_Omelette.jpg
modified:   frontend/src/assets/images/Spicy_Beef.jpg
modified:   frontend/src/assets/images/Stir-Fried_Baby_Bok_Choy.jpg
modified:   frontend/src/assets/images/Stir-Fried_Green_Beans.jpg
modified:   frontend/src/assets/images/Summer_Wrap.jpg
modified:   frontend/src/assets/images/TG_Burger.jpg
modified:   frontend/src/assets/images/TG_Cubano.jpg
modified:   frontend/src/assets/images/Tomato_and_Egg_Rice_Bowl.jpg
modified:   frontend/src/assets/images/Torta.jpg
modified:   frontend/src/assets/images/Tortillas_Q.jpg
modified:   frontend/src/assets/images/Vegetable_Chow_Mein.jpg
modified:   frontend/src/assets/images/Vegetable_Fried_Rice.jpg
modified:   frontend/src/assets/images/Vegetables_Hand-Ripped_Noodle_with_Special_Sauce.jpg
modified:   frontend/src/assets/images/Veggie_Burrito.jpg
modified:   frontend/src/assets/images/Veggie_Taco.jpg
modified:   frontend/src/assets/images/Wonton_Soup.jpg
modified:   frontend/src/assets/images/Wonton_with_Chili_Oil.jpg
modified:   frontend/src/assets/images/Zucchnini.jpg
modified:   frontend/src/assets/images/all_american_burger.jpg
new file:   frontend/src/assets/images/chille_cheese_fries.jpg
new file:   frontend/src/assets/images/i.jpg

xulean committed:
Merge branch 'munch_task87' of https://github.com/kelseyp/munch into munch_task87

xulean committed:
change brownie

austinhedrick548 committed:
Re-init'd db, removed excess pics

austinhedrick548 committed:
Merge branch 'develop' into munch_task87

austinhedrick548 reviewed (approved):
Works good, just doesn't load in codespaces for some reason

NathanaelPaulus reviewed (approved):
looks good
